helix-term/src/commands.rs

```diff
@@ -581,21 +581,29 @@ fn goto_first_nonwhitespace(cx: &mut Context) {
 }
 
 fn goto_window(cx: &mut Context, align: Align) {
+    let count = cx.count.map(|_| cx.count() - 1);
```

Comment:
```suggestion
    let count = cx.count();
```
Using this here would be better and we can do the adjustment below.